scripts/start_test_server.py

```diff
@@ -0,0 +1,38 @@
+import errno
+import sys
+
+from server.common.config.app_config import AppConfig
+from server.common.config.server_config import ServerConfig
+from server.tests.unit import TestServer
+
+PROJECT_ROOT: str = sys.argv[1]
+
+TEST_CONFIG_FILE = f"{PROJECT_ROOT}/client/__tests__/e2e/test_config.yaml"
+DATASET_DATAPATH = f"{PROJECT_ROOT}/example-dataset"
+DATASET = "pbmc3k.h5ad"
+
+# Create AppConfig and update ServerConfig for test server instance
+app_config: AppConfig = AppConfig()
+app_config.update_from_config_file(TEST_CONFIG_FILE)
+server_config: ServerConfig = app_config.server_config
+
+app_config.update_server_config(
+                app__flask_secret_key="SparkleAndShine",
+                multi_dataset__dataroot=DATASET_DATAPATH
+            )
```

Comment:
needs reformatting

Reply:
is this still relevant?

Reply:
I guess if lint passes then i have no complaints, just curious if it does with that alignment 🤔